SickBeard.py

```diff
@@ -349,6 +349,16 @@ def start(self):
         if self.consoleLogging:
             print "Starting up SickRage " + sickbeard.BRANCH + " from " + sickbeard.CONFIG_FILE
 
+        # Clean up after update
+        if sickbeard.GIT_NEWVER:
+            toclean = os.path.join(sickbeard.CACHE_DIR, 'mako')
+            for root, dirs, files in os.walk(toclean, topdown=False):
+                for name in files:
+                    os.remove(os.path.join(root, name))
+                for name in dirs:
+                    os.rmdir(os.path.join(root, name))
+            sickbeard.NEWVER = 0
```

Comment:
Ah. I missed that at 3am :D

Will correct it later today, should be GIT_NEWVER

-----Original Message-----
From: "Gaëtan Muller" <notifications@github.com>
Sent: ‎9/‎12/‎2015 9:00 AM
To: "SiCKRAGETV/SickRage" <SickRage@noreply.github.com>
Cc: "Nils" <nils@familievogels.nl>
Subject: Re: [SickRage] Adding mako cache cleaner upon startup of new version(#2482)

In SickBeard.py:
> @@ -349,6 +349,16 @@ def start(self):
>          if self.consoleLogging:
>              print "Starting up SickRage " + sickbeard.BRANCH + " from " + sickbeard.CONFIG_FILE
>  
> +        # Clean up after update
> +        if sickbeard.GIT_NEWVER:
> +            toclean = os.path.join(sickbeard.CACHE_DIR, 'mako')
> +            for root, dirs, files in os.walk(toclean, topdown=False):
> +                for name in files:
> +                    os.remove(os.path.join(root, name))
> +                for name in dirs:
> +                    os.rmdir(os.path.join(root, name))
> +            sickbeard.NEWVER = 0
GIT_NEWER?
—
Reply to this email directly or view it on GitHub.